<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogo_ConsultaDatosTabla</class>
 <widget class="QDialog" name="Dialogo_ConsultaDatosTabla">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sqlite3 Consulta de Datos</string>
  </property>
  <widget class="QLabel" name="lbl_nombre_base">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de la base de datos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nombre_tabla">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre tabla:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nombre_base_datos">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nombre_tabla">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_consultar_datos">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Consultar datos</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbl_datos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>371</width>
     <height>171</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
